### PR TITLE
Updates for Online Store System v1.0 multiple vulnerabilities

### PR DESCRIPTION
**Pull request by lcashdol**

### CODE REVIEW

File: 2019/8xxx/CVE-2019-8292.json [hunk 1]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2019/8xxx/CVE-2019-8292.json]: File name CVE ID and JSON CVE ID do not match.

Comment (lcashdol) [reply 2019/8xxx/CVE-2019-8292.json]: Done.

**Comment by lcashdol** [comment]
Hi, how does this look?

**Comment by cve-team** [comment]
It looks like a couple of the descriptions are missing version information (i.e. v1.0). CVE-2019-8288 and CVE-2019-8290 were the only ones.

**Comment by cve-team** [comment]
Ok thanks! These have been populated and should be up on cve.mitre.org within an hour or so.

**Pull request merged**